File: python/roman-numerals/roman_numerals.py
arabic_values = [1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1]
roman_numerals = ["M", "CM", "D", "CD", "C", "XC", "L", "XL", "X", "IX", "V", "IV", "I"]


def roman(number):

    roman_numeral = ""
    for i in range(len(arabic_values)):
        while number >= arabic_values[i]:
            roman_numeral += roman_numerals[i]
            number -= arabic_values[i]

    return roman_numeral

File: csharp/roman-numerals/RomanNumerals.cs

public static class RomanNumeralExtension
{
    private static int[] ARABIC_NUMERALS = new int[]{1000, 900, 500, 400, 100, 90, 50, 40, 10, 9, 5, 4, 1};
    private static string[] ROMAN_NUMERALS = new string[]{"M", "CM", "D", "CD", "C", "XC", "L", "XL", "X", "IX", "V", "IV", "I"};
    
    public static string ToRoman(this int value)
    {
        var roman = "";
        for (int i = 0; i < ARABIC_NUMERALS.Length; i++)
        {
            while (value >= ARABIC_NUMERALS[i])
            {
                roman += ROMAN_NUMERALS[i];
                value -= ARABIC_NUMERALS[i];
            }
        }

        return roman;
    }
    
}